discovery-infra/tests/test_day2.py

```diff
@@ -0,0 +1,40 @@
+import day2
+import pytest
+from junit_report import JunitTestSuite
+from tests.base_test import BaseTest
+from tests.config import ClusterConfig, global_variables
+from types import SimpleNamespace
+
+
+class TestDay2(BaseTest):
+    @pytest.fixture
+    def new_cluster_configuration(self):
+        return self.override_cluster_configuration()
+
+    def override_cluster_configuration(self):
+        config = ClusterConfig()
+        config.cluster_id = global_variables.cluster_id
+        return config
+
+    # Install day1 cluster and deploy day2 nodes (cloud flow).
+    # Or, deploy day2 nodes on an installed cluster if CLUSTER_ID env var is specified.
+    @JunitTestSuite()
+    def test_deploy_nodes_cloud(self, cluster):
```

Comment:
Rename it to more unique name
e.g `test_day2_deploy_nodes_cloud`

Reply:
done